hack/.ci/run-e2e-gke.sh

```diff
@@ -36,6 +36,7 @@ if [ -z ${ARTIFACTS+x} ]; then
   exit 2
 fi
 
+SO_NODECONFIG_DISK_TYPE=${SO_NODECONFIG_DISK_TYPE:-"nvme"}
```

Comment:
nit: you could check for supported disk types to fail early instead of on kubectl command

Reply:
I changed it to accepting path to NodeConfig instead suffix. This allows to specify custom NodeConfig suitable for given setup, which may be defined alongside kubernetes cluster spec.